packages/arb-token-bridge-ui/tsconfig.json

```diff
@@ -15,7 +16,13 @@
       "@/images/*": ["./public/images/*"],
       "@/icons/*": ["./public/icons/*"],
       "@/token-bridge-sdk/*": ["./src/token-bridge-sdk/*"]
-    }
+    },
+    "plugins": [
+      {
+        "name": "next"
+      }
+    ],
+    "strictNullChecks": true
```

Comment:
Similar comment - can we move this change, and it's corresponding impacting changes if any, to another PR?

Reply:
<img width="550" alt="image" src="https://github.com/user-attachments/assets/303c1c44-55e0-48fd-ab63-8ae8d642f941">